nixio/data_frame.py

```diff
@@ -68,7 +78,13 @@ def append_column(self, column, name, datatype=None):
         self.write_direct(farr)
 
     def append_rows(self, data):
-        # In Python2, the data supplied must be iterable (not np arrays)
+        """
+        Append a new row to the DataFrame
+        In Python2, the data supplied must be iterable (not np arrays)
```

Comment:
What happens in Python 2 with np.arrays?

Reply:
Could not find any problems with Python2. Will delete the sentence.

On the other hand, I am thinking if the DataFrame.dtype property should return the DataFrame._h5group.group["data"].dtype (np.dtype) instead of the raw list of dtypes now. 
As it is more helpful to create np arrays with same compound data type before writing and appending new rows.

Reply:
I guess the original problem I had in mind is that the np arrays in python 2 have less of dynamic typing and probably there was some problem when I previously test it. But if the array is created with compound type, there is no problem